public/app/plugins/panel/barchart/BarChartPanel.tsx

```diff
@@ -321,7 +328,33 @@ export const BarChartPanel = ({ data, options, fieldConfig, width, height, timeZ
       height={height}
     >
       {(config) => {
-        if (oldConfig.current !== config) {
+        if (showNewVizTooltips && options.tooltip.mode !== TooltipDisplayMode.None) {
+          return (
+            <TooltipPlugin2
+              config={config}
+              hoverMode={
+                options.tooltip.mode === TooltipDisplayMode.Single ? TooltipHoverMode.xOne : TooltipHoverMode.xAll
+              }
+              render={(u, dataIdxs, seriesIdx, isPinned, dismiss, timeRange2) => {
```

Comment:
```suggestion
              render={(_u, dataIdxs, seriesIdx, isPinned, _dismiss, _timeRange2) => {
```

Reply:
we don't do this anywhere else. also, why?

Reply:
weird. i have never seen that. usually that's a prefix used for internal/private methods/props.

what's the value of doing this if your IDE will grey it out anyhow?

<img width="447" alt="image" src="https://github.com/grafana/grafana/assets/43234/20430a81-0c12-4b46-b5db-5104b16f9628">

Reply:
i did a search in the entire grafana codebase and only found 36 instances of this in 15 files. i would say this is definitely not a convention used in this codebase :rofl: 

<img width="981" alt="image" src="https://github.com/grafana/grafana/assets/43234/ac53c9f3-c96c-416c-9da3-033060df4219">